acc.config.json

```diff
@@ -1,10 +1,12 @@
 {
   "logo": {
-    "url": null,
-    "width": 0,
-    "height": 0,
-    "smallWidth": 0,
-    "smallHeight": 0
+    "url": "/assets/logos/logo-weesp.svg",
+    "width": "unset",
+    "height": "100%",
+    "maxHeight": "67px",
+    "smallWidth": "unset",
+    "smallHeight": "100%",
+    "maxSmallHeight": "67px"
```

Comment:
Can't we solve all this with CSS? If we start going down the route of having CSS rule values in our configuration, then it'll become a mess fairly quickly.

Reply:
Probably we can, however I 'continued' on the work I merged and this is IMHO out of scope for SIG-2464.

Reply:
The scope of SIG-2464 is the logo, so I wouldn't say it's out of scope. Let's try to solve this CSS-side to limit the amount of config lines required for just an image.

Reply:
I totally agree, will put this on resolved since we agreed to file a new ticket for this task,